langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -139,6 +197,10 @@ public isolated function includes(string str, string substr, int startIndex = 0)
 
 # Tests whether a string starts with another string.
 #
+# ```ballerina
+# "Welcome to the Ballerina programming language".startsWith("Welcome") ⇒ true
```

Comment:
How about adding an example resulting in `false`?

Reply:
I felt this case ignorable. Hence not covered.